content/docs/command-reference/params/index.md

```diff
@@ -28,14 +28,17 @@ YAML 1.2, JSON, TOML, or [Python](#examples-python-parameters-file) _parameters
 file_. Multiple parameter dependencies can be specified from one or more
 parameters files.
 
+💡 Parameters can also be used directly in `dvc.yaml` itself. See the
+[Parameterize stages](#example-parameterize-stages-in-dvcyaml) example below.
+
 The default parameters file name is `params.yaml`. Parameters should be
 organized as a tree hierarchy inside, as DVC will locate param names by their
 tree path. Parameters files have to be manually written, or generated, and these
 can be versioned directly with Git.
 
 Supported parameter _value_ types are: string, integer, float, and arrays. DVC
 itself does not ascribe any specific meaning for these values. They are
-user-defined, and serve as a way to generalize and parametrize an machine
+user-defined, and serve as a way to generalize and parameterize an machine
```

Comment:
Is `parameterize` preferred compared to `parametrize`?

Reply:
I did a quick trends check on this and decided for that, yes. But there's no clear consensus out there.

I personally like "parameterize" better: parameter + iz(e|ation). Without the extra 'e' it seems closer to "parametric" which is slightly different.

Reply:
> Without the extra 'e', the root seems closer to "parametric", which is slightly different.

Which is what is it. Anyway, it [looks like `parameterize` is dominant form in US-English](https://en.wiktionary.org/wiki/parameterize).